eg/Classes/MainFrame/__init__.py

```diff
@@ -521,12 +521,45 @@ def GetPanelDirection(self, panel):
         pos = info.find(";dir=") + 5
         return info[pos]
 
+    def SetWindowStyleFlag(self, *args, **kwargs):
+        """
+        Changes the main frame style depending on the display of the tray icon.
+
+        sets the stlye flags fo the minimize button will be grayed out if the
```

Comment:
ooops hehe!. I have bad arthritis. And my fingers get very lazy to that shift key.